x-pack/filebeat/module/gsuite/saml/config/pipeline.js

```diff
@@ -0,0 +1,60 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+var saml = (function () {
+    var processor = require("processor");
+
+    var categorizeEvent = function(evt) {
+        evt.Put("event.type", ["info"]);
```

Comment:
Still not entirely convinced that this should be `info`--for `authentication` events the `type` values basically describe lifecycle events to an authentication flow (i.e. `start`/`end` which are basically log in/log out) with a generic type for informational data that doesn't relate directly to the authentication lifecycle (i.e. `info`).

It seems to me that these SAML auth events are basically describing whether an `authentication` log in (i.e. the start of an authentication) succeeded or not, in which case I'd argue that this is `category: authentication, type: start, outcome: success/failure`. But if:

1. I'm misunderstanding the scope of what these SAML auth events can be or
2. You have a really strong preference for this being `info` then feel free to keep this `info`

When constructing these categories though, I try and ask, "what would someone want to query with these and how would they go about doing so?" I think that if someone were trying to get the login failures for a system they'd ask something like, "show me the authentication events where they failed at the start" (i.e. failed to log in).

Reply:
I do not have a strong opinion on using `info`, if as you mention we understand the `start`, `end`, `info` types applying on the whole `authentication` life cycle, then it makes more sense to put `start`. I interpreted them more as the login attempt itself, `start` the attempt and then `end`s, but since they covered the complete attempt I thought `info` was a good fit. Now that you mention that, and since probably it is the common understanding, I will change it to use `start` :+1: